src/redux/selectors/reportingRateSummary/parseTableData.js

```diff
@@ -1,4 +1,57 @@
+import i18n from '@dhis2/d2-i18n'
+import isEmpty from 'lodash.isempty'
+
+const nameColumnDisplayName = i18n.t('Name')
+const rowIndexesToRead = [
+    1, // Org unit name
+    4, // column values
+    5,
+    6,
+    7,
+    8,
+]
+
 export default function parseTableData(state) {
```

Comment:
Selectors should always start with `get` (also part of the [result of the discussion](https://github.com/dhis2/notes/pull/19) we had last week friday and yesterday). Also `parse` has a different meaning than `build` or sth like that, so it might be even more confusing to use `parse` here (as well as for the other selectors).